### PR TITLE
Added event title styling to Event component

### PR DESCRIPTION
**Pull request by vjy02**
Let me know of any issues.

### CODE REVIEW

File: src/routes/Events.tsx [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
-};
+  return (
+    <div className="w-screen max-w-full">
 
-export default Events;
+        {/* Title */}
+        <div className="flex flex-col items-center md:mt-10">
+          <div className="z-0 absolute top-[10%] left-[-37%] text-9xl font-black md:left-[-12%] md:text-[256px] overflow-hidden whitespace-nowrap" style={{
+            color: 'transparent',
+            WebkitTextStrokeWidth: '1px',
```
Comment (e3lo) [inline src/routes/Events.tsx:9]: Change text stroke to be 2px on md screen sizes and 1px on mobile

File: src/routes/Events.tsx [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+            WebkitTextStrokeColor: '#767676'
+          }}>
+            Events / Initiatives
```
Comment (e3lo) [inline src/routes/Events.tsx:12]: Extract all text as props

**Review by e3lo — CHANGES_REQUESTED** [review]
Looks really nice just update the stroke width and props

**Comment by vjy02** [comment]
@e3lo I updated the text as props but have it inside main.tsx atm, not sure if that is good or is it fine to just be there for now?

**Comment by e3lo** [comment]
@vjy02 Oops I didn't realise, move the event title code into a new folder in features so it should be src/features/TitleHero/TitleHero.tsx
Then in Events.tsx import TitleHero and use it.

Essentially Events.tsx is the new root.tsx for the events page

**Comment by vjy02** [comment]
@e3lo All done lmk if it needs any more changes

**Review by e3lo — APPROVED** [review]

**Pull request merged**